tests/python/driver/tvmc/test_autotuner.py

```diff
@@ -207,3 +209,27 @@ def test_autotune_pass_context(mock_pc, onnx_mnist, tmpdir_factory):
     # AutoTVM overrides the pass context later in the pipeline to disable AlterOpLayout
     assert mock_pc.call_count == 2
     assert mock_pc.call_args_list[0][1]["opt_level"] == 3
+
+
+def test_filter_tasks_valid():
+    filter_tasks(list(range(10)), "list") == ([], True)
+    filter_tasks(list(range(10)), "help") == ([], True)
+    filter_tasks(list(range(10)), "all") == ([0, 1, 2, 3, 4, 5, 6, 7, 8, 9], False)
+    filter_tasks(list(range(10)), "5") == ([5], False)
+    filter_tasks(list(range(10)), "1-5") == ([1, 2, 3, 4, 5], False)
+    filter_tasks(list(range(10)), "-5") == ([0, 1, 2, 3, 4, 5], False)
+    filter_tasks(list(range(10)), "6-") == ([6, 7, 8, 9], False)
+    filter_tasks(list(range(10)), "0,1-3,all") == ([0, 1, 2, 3, 4, 5, 6, 7, 8, 9], False)
+    filter_tasks(list(range(10)), "0,4-5,9,list") == ([0, 4, 5, 9], True)
+
+
+@pytest.mark.xfail
```

Comment:
XFail is normally used to mark a test that is "unexpectedly failing", which doesn't seem like the case here. If the test suddenly starts passing it won't be flagged in CI. I think it would be better to rewrite this test using `pytest.raises` e.g. something like:
```
@pytest.mark.parameterize("value,err_msg", [("10", "my expected error message"), ("5,10", "my expected error message 2"), ...])
def test_filter_tasks_invalid(value, err_msg):
    with pytest.raises(AssertionError, match=err_msg):
        filter_tasks(list(range(10)), value)
    
```